templates/common-partials/script/appliedfilters.hbs

```diff
@@ -1,7 +1,13 @@
-ANSWERS.addComponent('AppliedFilters', {
-  container: '#js-answersAppliedFilters',
-  {{#if verticalKey}}
-    verticalKey: '{{{verticalKey}}}',
-  {{/if}}
-  ...{{{ json componentSettings.AppliedFilters }}}
-});
\ No newline at end of file
+ANSWERS.addComponent('AppliedFilters', 
+  {{#assign}}[
+  {
+    "container": "#js-answersAppliedFilters"
+    {{#if verticalKey}}
+    ,
+    "verticalKey": "{{{verticalKey}}}"
+    {{/if}}
+  },
+  {{{json componentSettings.AppliedFilters}}}
+]
```

Comment:
would it be weird to smoosh lines 11-13 together? and also lines 1-2?

Reply:
theres some weird behavior so I'm gonna leave it unchanged for now, but if it matters I can definitely make the change.

Reply:
what is the weird behavior?

Reply:
extra new lines, but it doesnt make a difference if i change all of them, bc then its consistent